src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -184,7 +184,11 @@ private static void destroy(String container, Launcher launcher, Node node, EnvV
             container = dockerClient.run(env, step.image, step.args, ws, volumes, volumesFromContainers, envReduced, dockerClient.whoAmI(), /* expected to hang until killed */ "cat");
             final List<String> ps = dockerClient.listProcess(env, container);
             if (!ps.contains("cat")) {
-                listener.error("The container started but didn't run the expected command. Please double check your ENTRYPOINT does execute the command passed as docker run argument. See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#entrypoint for entrypoint best practices.");
+                listener.error(
+                    "The container started but didn't run the expected command. " +
+                        "Please double check your ENTRYPOINT does execute the command passed as docker run argument, " +
+                        "as required by official docker images (see https://github.com/docker-library/official-images#consistency for entrypoint best practices).\n" +
```

Comment:
How about "See ... for expected ENTRYPOINT behavior"? Docker never calls it a "best practice", just a consistency requirement for official images. And public documentation calls behavior not conferment to this "the best use" for ENTRYPOINT. So, I'd shy away from language that implies anything about the quality of the images used.